Samples/Client/Unity/Assets/Scripts/ControlScript.cs

```diff
@@ -221,7 +221,7 @@ void Update()
 
         cameraTransformBody =
            "{" +
-           "  \"type\":\"camera-transform-stereo\"," +
+           "  \"type\":\"camera-transform-stereo-prediction\"," +

```

Comment:
now this makes the client incompatible with old servers, correct? old servers will never get stereo transform updates if the client is only sending prediction-enabled updates

Reply:
Oh yes, I prefer this way since using the *try-parse* method seems a little bit hack to me.